mono/mini/driver.c

```diff
@@ -2348,15 +2349,25 @@ mono_main (int argc, char* argv[])
 			mono_runtime_install_custom_handlers_usage ();
 			return 0;
 		} else if (strncmp (argv [i], "--response=", 11) == 0){
-			gchar *text;
-			gsize len;
-			
-			if (!g_file_get_contents (&argv[i][11], &text, &len, NULL)){
+			gchar *response_content;
+			gchar *response_options;
+			gsize response_content_len;
+
+			if (!g_file_get_contents (&argv[i][11], &response_content, &response_content_len, NULL)){
 				fprintf (stderr, "The specified response file can not be read\n");
 				exit (1);
 			}
-			mono_parse_options (text, &argc, &argv, FALSE);
-			g_free (text);
+
+			response_options = response_content;
+
+			// Check for UTF8 BOM in file and remove if found.
+			if (response_content_len > 3 && response_content [0] == '\xef' && response_content [1] == '\xbb' && response_content [2] == '\xbf') {
```

Comment:
Shouldn't it be >=3 not >3? i.e. an empty file?

Reply:
Thought about that as well, but its the same implementation as found in other places of the source code like, mono_config_parse_file_with_context, mono_domain_set_options_from_config and if file is empty things will still break, but agree that we could error case for file just including BOM mark.

Reply:
I think it should be  >=3 and gracefully handle an otherwise empty file.

Reply:
Will fix.